idl/response/foxsi_get_shutter_transmission.pro

```diff
@@ -0,0 +1,45 @@
+;+
+; NAME : get_foxsi_get_shutter_transmission
+;
+; PURPOSE : Returns the shutter transmission efficiency.
+;
+; SYNTAX : trans = get_foxsi_get_shutter_transmission()

```

Comment:
Ok this is a very minor quibble, but you can't actually call this with no arguments (shutter state is required)